.github/workflows/master-change.yml

```diff
@@ -33,6 +33,29 @@ jobs:
           token: ${{ secrets.NPM_TOKEN_GITHUB }}
           registry: https://npm.pkg.github.com
 
+  make-release:
+    needs: [npm-publish-package]
```

Comment:
This prevents the release from running unless the npm package succeeds. The npm package has a uniqueness requirement so if the version number is not changed, then it will fail and hence, so will the release. This should help us not make mistakes.